daemon/src/daemon.cpp

```diff
@@ -219,24 +224,22 @@ int main( int argc, const char* argv[] )
     cf_daemon.gmsa_account_name = CF_TEST_GMSA_ACCOUNT;
 
     std::cerr << "krb_files_dir = " << cf_daemon.krb_files_dir << std::endl;
-    //std::cerr << "cred_file = " << cf_daemon.cred_file <<  " (lease id: " << cred_file_lease_id
-             //   << ")" << std::endl;
+    // std::cerr << "cred_file = " << cf_daemon.cred_file <<  " (lease id: " << cred_file_lease_id
```

Comment:
good to remove @smhmhmd?

Reply:
The std:cerr can be added to write_log as well, also, those prints can be brought back.

Reply:
Shall I add all of these cerr logs to write_log?

Reply:
Please go ahead